modules/encrypt_seclib/src/Plugin/EncryptionMethod/PHPSecLibEncryption.php

```diff
@@ -13,7 +13,8 @@
  * @EncryptionMethod(
  *   id = "phpseclib",
  *   title = @Translation("PHP Secure Communications Library (phpseclib)"),
- *   description = "Uses the <a href='http://phpseclib.sourceforge.net/'>phpseclib</a> library. This method is only preferable if you cannot install mcrypt."
+ *   description = "Uses the <a href='http://phpseclib.sourceforge.net/'>phpseclib</a> library. This method is only preferable if you cannot install mcrypt.",
+ *   key_type = {"aes_encryption"}
```

Comment:
Yes, I created "aes_encryption" key type (see other PR). The PHPSecLib library uses AES, so should use that one, in order to only allow keys of 128, 192 or 256 bits. The generic "encryption" key type allows more sizes, which would be a problem if used with PHPSecLib.

Either way, this class is heading out the door anyway :)